.appveyor.yml

```
@@ -23,7 +23,7 @@ install:
   - pyinstaller --name oj --onefile onlinejudge\_implementation\main.py
 
 test_script:
-  - python setup.py test
+  - pytest tests/ -v
```

```suggestion
  - ps: pytest -v $(dir tests/*.py | % {$_.FullName})
```

1. で解決できればそれが最良で、AppVeyor では PowerShell が使えるのでできるはずです。
手元で試した限りだとこれで動きます。